src/main/java/uk/gov/hmcts/divorce/divorcecase/model/NoResponseJourneyOptions.java

```diff
@@ -28,4 +32,34 @@ public class NoResponseJourneyOptions {
             access = {DefaultAccess.class}
     )
     private YesOrNo noResponsePartnerHasReceivedPapers;
+
+    @CCD(
+        label = "I have a new postal or email address",
+        access = {DefaultAccess.class},
+        typeParameterOverride = "NoResponseNewEmailAndPostalAddress"
+    )
+    private NoResponseNewEmailAndPostalAddress noResponseNewEmailAndPostalAddress;
+
```

Comment:
Is this field used for the applicant or their partner? The field name and label suggests the new details are for the applicant themselves.

If these are new details for their partner, could we clarify in the naming of the field and the label to prevent future confusion?  Perhaps NoResponseNewPartnerEmailAndPostalAddress?